### PR TITLE
MAT-8732 admin api to restart validations

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-8732](https://jira.cms.gov/browse/MAT-8732)
(Optional) Related Tickets:

### Summary
admin api to restart validations

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by sb-cecilialiu — DISMISSED** [review]

File: src/main/java/cms/gov/madie/measure/resources/AdminController.java [hunk 1]
```diff
@@ -123,6 +124,60 @@ public ResponseEntity<MeasureTestCaseValidationReportSummary> validateAllMeasure
...
+                              testCase,
+                              accessToken,
+                              ModelType.valueOfName(measure.getModel()));
+                        } else if (TestCaseValidationStatus.VALIDATING
+                            .toString()
+                            .equalsIgnoreCase(testCase.getValidationStatus())) {
+                          measureRepository.setValidationStatusToPending(
+                              testCase.getId(), measure.getId());
+                          // submit the test case after updating its status
+                          testCaseValidationService.submitOnSaveValidationTask(
```
Comment (jkotanchik-SB) [inline src/main/java/cms/gov/madie/measure/resources/AdminController.java]: The status setter methods return the modified Measure when successful and `null` otherwise. It might be helpful to verify the returned value is not null prior to submitting the Test Case for validation.
```suggestion
                          Measure updatedMeasure = measureRepository.setValidationStatusToPending(
                              testCase.getId(), measure.getId());
                          if (updatedMeasure == null) break; // Issue updating status to Pending.
                          // submit the test case after updating its status
                          testCaseValidationService.submitOnSaveValidationTask(
```

Comment (sb-prateekkeerthi) [reply src/main/java/cms/gov/madie/measure/resources/AdminController.java]: thank you, I have updated it

**Review by jkotanchik-SB — APPROVED** [review]
